bitnami/redis/README.md

```diff
@@ -258,6 +258,7 @@ The following table lists the configurable parameters of the Redis<sup>TM</sup>
 | `sentinel.failoverTimeout`                            | Timeout for performing a election failover                                                                                                          | `18000`                                                 |
 | `sentinel.parallelSyncs`                              | Number of parallel syncs in the cluster                                                                                                             | `1`                                                     |
 | `sentinel.port`                                       | Redis<sup>TM</sup> Sentinel port                                                                                                                    | `26379`                                                 |
+| `sentinel.cleanWait`                                  | Seconds to wait when cleaning nodes IPs                                                                                                             | `5`                                                     |
```

Comment:
I'd use (more descriptive parameter name and follows same naming as probes):
```suggestion
| `sentinel.cleanTimeoutSeconds`                        | Seconds to wait before cleaning nodes IPs                                                                                                             | `5`                                                     |
```

Reply:
This is not a timeout, it is just a nice wait before making a new request. There is nothing that needs to happen in a specified time frame.

Reply:
I updated the description.

Reply:
Then, use `cleanDelaySeconds` (so it's more alike to `xxxProbe.initialDelaySeconds`)

Reply:
ok